examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>In general, can we make the example more lean with focus on the dataloader + training? </p><p> </p><p> </p><p>It seems that we can use merlin.core -&gt; we have a function to download and extract a dataset:</p><pre class="ql-syntax" spellcheck="false">from merlin.core.utils import download_file
</pre><p> </p><p>Can we use this? See</p><p>https://github.com/NVIDIA-Merlin/NVTabular/blob/main/examples/getting-started-movielens/01-Download-Convert.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>absolutely, that is a great point, switched to using <code>download_file</code></p>